src/runtime/current_thread/runtime.rs

```diff
@@ -56,56 +61,60 @@ impl Runtime {
         Ok(runtime)
     }
 
-    /// Blocks and runs the given future.
+    /// Runs the provided future, blocking the current thread until the future
+    /// completes.
+    ///
+    /// This function can be used to synchronously block the current thread
+    /// until the provided `future` has resolved either successfully or with an
+    /// error. The result of the future is then returned from this function
+    /// call.
+    ///
+    /// Note that this function will **also** execute any spawned futures on the
+    /// current thread, but will **not** block until these other spawned futures
+    /// have completed.
```

Comment:
It will be useful to say what does happen to any remaining spawned futures after this function returns.